core/src/test/scala/unit/kafka/security/auth/SimpleAclAuthorizerTest.scala

```diff
@@ -271,4 +359,46 @@ class SimpleAclAuthorizerTest extends ZooKeeperTestHarness {
 
     acls
   }
+
+  /**
+    * https://github.com/junit-team/junit/wiki/Multithreaded-code-and-concurrency
+    *
+    * To use this you pass in a Collection of Runnables that are your arrange\act\assert test on the SUT,
+    * they all run at the same time in the assertConcurrent method; the chances of triggering a multithreading code error,
+    * and thereby failing some assertion are greatly increased.
+    */
+  private def assertConcurrent(message: String, runnables: List[Runnable], maxTimeoutSeconds: Int) {
```

Comment:
It would be better to take functions instead of runnables. Also, maybe use timeout in ms since that's more common in Kafka.